### ISSUE TITLE
[Feature request] Offlines as idle

### ISSUE THREAD

**Issue by nelim** [vitalidze/traccar-web#374]
Can you count offlines for more than specific time(60s) for car as idles ?

My idea is to count this as stop, otherwise if I stop and poweroff gps at night and continue my drive at the morning it counts that like one long drive for hours on the report.

Thanks in advance

**Comment by vitalidze** [comment]
I think the 'Drives and stops' report should be improved to not count the 'offline' as the movement even if last position can be considered as movement. This should be improved in some newer versions.

**Comment by nelim** [comment]
Yep, my 50cents on that issue confirm that last position speed is the problem:
I've done some tests and it if i set speed to 0(or what is set as options for carIdlespeed) on all OFFLINE positions from events table, everything looks like OK on my Moving/Stops report;

Problem is only when device is shutted down before speed drops to zero, maybe an easy solution is to get only idleTime after that OFFLINE position_id and count is as stop, even if speed of that offline positions is >= 0

my current solution is:

`
<mysql>
update positions set speed = 0 where id in (select position_id from events where type = "OFFLINE");
`
and ReportDS are OK, maybe speed and avg/speed calculations will not be that accurate, but that is fine for now.

Good luck :) :+1:

**Comment by vitalidze** [comment]
Thank you. I will think about it.
